src/blocks/subscribe/edit.js

```diff
@@ -0,0 +1,156 @@
+/* global newspack_newsletters_blocks */
```

Comment:
Can be removed, it's not used.

Reply:
It is used on line 27:

```js
const settingsUrl = window.newspack_newsletters_blocks.settings_url;
```

Reply:
Oh, my bad!

Reply:
Here, `window` global is used. If it was `newspack_newsletters_blocks.settings_url` then the global declaration at the top of the file would be warranted.

Reply:
Done in cf2762d377781d84e73b5914393afb48fc878f7e